client/src/components/App.js

```diff
@@ -30,7 +30,10 @@ const App = (props) => {
       <TopBar user={currentUser} />
       <Switch>
         <Route exact path="/">
-          <h2 className="welcome-message">Welcome to the Movies Review App</h2>
+          <h2 className="welcome-message">Reel Reviews</h2>
+          <h4 className="welcome-message">Take a seat, enjoy the space.</h4>
+          <img className="logo" src={three}/>
+          <p className="authors">Developed and Designed by: Curt, Jose, and Solomon</p>
```

Comment:
lines 33-36 should all be inside of a WelcomeMessage component. Refactor this so that said component is rendered at the root path